packages/trader/src/Modules/CFD/Containers/cfd-server-error-dialog.tsx

```diff
@@ -1,9 +1,18 @@
-import PropTypes from 'prop-types';
 import React from 'react';
 import { Dialog } from '@deriv/components';
 import { localize, Localize } from '@deriv/translations';
 import { connect } from 'Stores/connect';
 
+type TCFDServerErrorDialog = {
+    clearCFDError?: () => void;
+    disableApp?: () => void;
+    enableApp?: () => void;
+    error_message: string;
```

Comment:
i suggest this because error_message is also a non-required field (the component can be called with no props):
```suggestion
    error_message?: string;
```